src/help/zaphelp/contents/cmdline.html

```diff
@@ -9,6 +9,14 @@
 <BODY BGCOLOR="#ffffff">
 <H1>Command Line</H1>
 
+To run ZAP via the command line, you will need to locate the `zap.sh` script. On Mac, this will be at:<br/>
+<pre>/Applications/OWASP\ ZAP.app/Contents/Java/zap.sh</pre>
+On Linux, it will be at<br/>
+<pre>/full/path/to/ZAP/zap.sh</pre>

```

Comment:
Not sure where it would be found on Windows...? Don't have a machine to test with, unfortunately.